bin/pycbc_make_skymap

```diff
@@ -381,22 +381,26 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
             max_bin = peak_bin + window_bins + 1
             min_bin = peak_bin - window_bins
 
-        followup_data[ifo]['snr_series'] = \
+        skyloc_data[ifo]['snr_series'] = \
                 snr_series[min_bin:max_bin].astype(np.complex64)
 
-    kwargs = {'psds': {ifo: followup_data[ifo]['psd']
-                       for ifo in ifos + subthreshold_ifos},
+    kwargs = {'psds': {ifo: skyloc_data[ifo]['psd'] for ifo in ifos},
               'low_frequency_cutoff': f_low,
               'high_frequency_cutoff': f_upper,
-              'followup_data': followup_data,
+              'skyloc_data': skyloc_data,
               'channel_names': channel_names}
 
-    doc = live.CandidateForGraceDB(ifos, coinc_results, **kwargs)
+    doc = live.CandidateForGraceDB(
+        trig_ifos,
+        trig_ifos,
```

Comment:
Why is the second argument not `ifos`?

Reply:
Ah, I guess because they do not contribute to the "significance", by looking at the updated docstring… Strictly speaking, none of the detectors contribute to the significance in this case.

Reply:
Any request for changes ?

Reply:
I suppose not.